charts/ndm/templates/02-daemonset.yaml

```diff
@@ -0,0 +1,148 @@
+{{- if .Values.ndm.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "ndm.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "ndm.name" . }}
+    chart: {{ template "ndm.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: ndm
+    openebs.io/component-name: ndm
+    openebs.io/version: {{ .Values.release.version }}
+spec:
+  updateStrategy:
+    type: "RollingUpdate"
+  selector:
+    matchLabels:
+      app: {{ template "ndm.name" . }}
+      release: {{ .Release.Name }}
+      component: ndm
+  template:
+    metadata:
+      labels:
+        app: {{ template "ndm.name" . }}
+        release: {{ .Release.Name }}
+        component: ndm
+        openebs.io/component-name: ndm
+        name: openebs-ndm
+        openebs.io/version: {{ .Values.release.version }}
+    spec:
+      serviceAccountName: {{ template "ndm.serviceAccountName" . }}
+      hostNetwork: true
+      containers:
+      - name: {{ template "ndm.name" . }}
+        image: "{{ .Values.ndm.image.registry }}{{ .Values.ndm.image.repository }}:{{ .Values.ndm.image.tag }}"
+        args:
+          - -v=4
+{{- if .Values.featureGates.enabled }}
+{{- if .Values.featureGates.GPTBasedUUID.enabled }}
+          - --feature-gates={{ .Values.featureGates.GPTBasedUUID.featureGateFlag }}
+{{- end}}
```

Comment:
There is a new additional feature gate `APIService`. That flag also need to be added here. 
Ref [openebs/openebs](https://github.com/openebs/openebs/blob/91b3968463dea4df861c7f2a15725a17624e4f35/k8s/openebs-operator.yaml#L490)